platform/ios/detect.py

```diff
@@ -153,3 +153,11 @@ def configure(env: "Environment"):
 
     if env["vulkan"]:
         env.Append(CPPDEFINES=["VULKAN_ENABLED"])
+
+    if env["opengl3"]:
+        env.Append(CPPDEFINES=["GLES_ENABLED", "GLES3_ENABLED"])
```

Comment:
`GLES_ENABLED` seems to be unnecessary and added by mistake in the macOS config:
```
$ rg GLES_ENAB
platform/uwp/detect.py
163:    env.Append(CPPDEFINES=["GLES_ENABLED", "GL_GLEXT_PROTOTYPES", "EGL_EGLEXT_PROTOTYPES", "ANGLE_ENABLED"])

platform/macos/detect.py
233:        env.Append(CPPDEFINES=["GLES_ENABLED", "GLES3_ENABLED"])

modules/openxr/openxr_api.cpp
70:#endif // GLES_ENABLED
```